### ISSUE TITLE
listSelect issues

### ISSUE THREAD

**Issue by nickmorales** [solgenomics/sgn#1261]
"Refresh" button is appended everytime modal is reopened.
When refreshed, the selection does not include the public lists (why is the dropdown being rebuilt in a different way when refreshed? the refresh should call the same function that originated the dropdown.) 

Maybe the refresh should be automated so that we don't need the button. The main purpose of the refresh was to account for displaying new lists, so maybe just refresh when the list modal is opened. 

![image](https://user-images.githubusercontent.com/7028512/29926604-e9816bc6-8e31-11e7-9b47-f410200eb154.png)

**Comment by nickmorales** [comment]
maybe just need to move the $("#select_list").append(list.listSelect("select_list", [ 'accessions' ], '', 'refresh')); lines out of the open_project_dialog() function and move them into the general document.ready section

**Comment by bellerbrock** [comment]
@nickmorales I just took a closer look. This is fixed in topic/fix_list_select_refresh if you want to review and merge

I changed `$("#select_list").append` to `document.getElementById("select_list").innerHTML =`, accomplishes the same thing by ensuring all child elements are replaced each time the modal is opened

And I simplified the refresh function so it just clears the parent div and runs the list select function again. That fixes the missing public lists problem. but creates problems of it's own - it requires that the div_name supplied to the listSelect function is the exact name of the actual parent div, and that the div contains nothing else. That's true now, of all the places where is list select function is called with the refresh function, but it's not true everywhere. 

I'm not sure if it's worth it, but we could simplify things further and eliminate the separate refreshListSelect funciton entirely if we refactored things so that listSelect is always called with the exact div_name.
